tachyon/crypto/commitments/pedersen/pedersen_unittest.cc

```diff
@@ -29,15 +29,24 @@ TEST_F(PedersenTest, CommitPedersen) {
       base::CreateVector(kMaxSize, []() { return math::bn254::Fr::Random(); });
 
   math::bn254::Fr r = math::bn254::Fr::Random();
-  math::bn254::G1AffinePoint commitment;
-  ASSERT_TRUE(vcs.Commit(v, r, &commitment));
+  math::bn254::G1AffinePoint cpu_commitment;
+  ASSERT_TRUE(vcs.Commit(v, r, &cpu_commitment));
```

Comment:
Do you think we need to test both cpu and gpu for cuda tests?

Reply:
I think so, because both are different code.

Reply:
Oh I was trying to say I think we only need cpu test when cuda is not enabled and we only need gpu test when cuda is enabled.

Reply:
Hmm, but `KZG` already tests like that, and since it doesn't take that long, i think it's okay!